podspec.yaml

```diff
@@ -1,3 +1,10 @@
+grow_version: ">=0.2.5"
```

Comment:
Can we omit `grow_version` and `home` line to keep this as minimal as possible? Then we're good to go. :) I sort of have a pet peeve with example/tutorial repos showing more than they need to, because then it makes it hard to figure out (and learn) exactly what's needed to achieve the task at hand.